README.md

```diff
@@ -0,0 +1,164 @@
+# *Alternate Bit Protocol Simulator*
+
+## FILES ORGANIZATION: Carleton University
+---
+### *Authors:*  
+---
+### Originally Developed by	: Dr. Cristina Ruiz Martin
+### Modified by				: Rajeev Kuamr Jandhyala and Abhinav Achanta
```

Comment:
give reference to GitHub accounts of authors so that one can look at their profiles to know more about the authors. see lecture pdf to see how you can implement this functionality.